src/uiclass.js

```diff
@@ -0,0 +1,84 @@
+import Store from './store.js';
+
+export default class UI {
+    static displayTodo = () => {
+      const todos = Store.getTodos();
+
+      todos.forEach((todo) => UI.addTodoToList((todo)));
+    }
+
+    static addTodoToList(todo) {
+      const todoList = document.querySelector('.todo-list');
+
+      const todoDiv = document.createElement('div');
+      todoDiv.classList.add('todo');
+
+      const completedButton = document.createElement('button');
+      completedButton.innerHTML = '<i class="fas fa-check"></i>';
+      completedButton.classList.add('toggle-btn');
+      todoDiv.appendChild(completedButton);
+
+      const newTodo = document.createElement('li');
+      newTodo.innerText = `${todo.description}`;
+      newTodo.id = `new-edit-${todo.index}`;
+      newTodo.classList.add('todo-item');
+      todoDiv.appendChild(newTodo);
+
+      const editTodo = document.createElement('input');
+      editTodo.value = `${todo.description}`;
+      editTodo.id = `todo-edit-${todo.index}`;
+      editTodo.style.display = 'none';
+      editTodo.classList.add('todo-item');
+      todoDiv.appendChild(editTodo);
+
+      const editButton = document.createElement('button');
+      editButton.innerHTML = '<i class="fas fa-edit"></i>';
+      editButton.id = `button-edit-${todo.index}`;
+      editButton.classList.add('edit-btn');
+      todoDiv.appendChild(editButton);
+
+      const trashButton = document.createElement('button');
+      trashButton.innerHTML = '<i class="fas fa-trash"></i>';
+      trashButton.classList.add('trash-btn');
+      todoDiv.appendChild(trashButton);
+      // Append to list
+      todoList.appendChild(todoDiv);
```

Comment:
You can create one div and concatenate all your code inside it, without appending piece by piece with that you will reduce your code size and respect more KISS concepts.

Reply:
- That's a smart suggestion thank you. 🤝